apps/omg_watcher_info/lib/omg_watcher_info/db/transaction.ex

```diff
@@ -32,6 +32,7 @@ defmodule OMG.WatcherInfo.DB.Transaction do
   @derive {Jason.Encoder, except: [:__meta__]}
   schema "transactions" do
     field(:txindex, :integer)
+    field(:txtype, :integer)
```

Comment:
I'd recommend an Enum here for clarity and type-safety instead of an integer.

Reply:
ignore this comment...just caught up with the rest of changes and discussion the PR.